### ISSUE TITLE
qt5 cmake config files broken

### ISSUE THREAD

**Issue by Optiligence** [msys2/MINGW-packages#5253]
```
CMake Error at D:/msys64/mingw64/lib/cmake/Qt5Gui/Qt5GuiConfig.cmake:15 (message):
  The imported target "Qt5::Gui" references the file

     "C:/building/msys32/mingw64/x86_64-w64-mingw32/lib/libglu32.a"

  but this file does not exist.  Possible reasons include:

  * The file was deleted, renamed, or moved to another location.

  * An install or uninstall procedure did not complete successfully.

  * The installation package was faulty and contained

     "D:/msys64/mingw64/lib/cmake/Qt5Gui/Qt5GuiConfigExtras.cmake"

  but not all the files it references.

Call Stack (most recent call first):
  D:/msys64/mingw64/lib/cmake/Qt5Gui/Qt5GuiConfigExtras.cmake:45 (_qt5_Gui_check_file_exists)
  D:/msys64/mingw64/lib/cmake/Qt5Gui/Qt5GuiConfigExtras.cmake:70 (_qt5gui_find_extra_libs)
  D:/msys64/mingw64/lib/cmake/Qt5Gui/Qt5GuiConfig.cmake:212 (include)
  D:/msys64/mingw64/lib/cmake/Qt5/Qt5Config.cmake:28 (find_package)
  CMakeLists.txt:70 (find_package)
```

**Comment by sickc** [comment]
Looks like qt5.12.3-1 CMake config wrongly referenced msys2 installation path as absolute "C:/building/msys32"

Re-installing into said path may work-around this issue.

**Comment by anlambert** [comment]
Same issue here (https://ci.appveyor.com/project/tulip-dev/tulip/build/job/mbqeq40tjhkhdddn#L1658).

**Comment by phoerious** [comment]
I noticed that a while ago and fixed it by editing out the C:/building of the Qt find module. I thought that was just another Msys2 packaging issue.

**Comment by jhasse** [comment]
Is this related to https://github.com/meta-qt5/meta-qt5/issues/108?

**Comment by lotharsm** [comment]
Yep, can confirm. See referenced issues above.

**Comment by Optiligence** [comment]
@Alexpux this is still an issue
the qt5 package cannot be used with a cmake build

**Comment by Alexpux** [comment]
I don’t have time to investigate it sorry

**Comment by Alexpux** [comment]
@Optiligence I think it is still issue with ````qt_configure.prf````, we fix ````pri & prl```` files in PKGBUILD and then replace paths using qtbinpatcher. But now it also propagated to cmake files too, so possible workarount to sed paths in ````cmake```` files and fix ````qtbinpather```` to do the same

**Comment by Alexpux** [comment]
@Optiligence first part https://github.com/msys2/MINGW-packages/commit/47b278a8a07dd15a80b20552c9988f50a25d0bb2

**Comment by Alexpux** [comment]
Finally fixed in 5.12.4-2 by https://github.com/msys2/MINGW-packages/commit/209d6b4d9eb7f9f8560dd88a9fe342d23346d6ef

**Comment by Optiligence** [comment]
Cool, thank you very much @Alexpux
